src/neuroglancer/ui/default_input_event_bindings.ts

```diff
@@ -87,6 +87,7 @@ export function getDefaultPerspectivePanelBindings() {
         {
           'at:mousedown0': {action: 'rotate-via-mouse-drag', stopPropagation: true},
           'at:shift+mousedown0': {action: 'translate-via-mouse-drag', stopPropagation: true},
+          'at:alt+mousedown2': {action: 'translate-annoation-via-mouse-drag', stopPropagation: true}
```

Comment:
Since it is the same for both perspective and slice view, you can instead specify this binding in the getDefaultRenderedDataPanelBindings() function.